x/metadata/types/events.go

```diff
@@ -22,26 +22,26 @@ const (
 	TKObjectAction string = "object-action"
 
 	// TLCategory is a string name for labels defining an object category.
-	TLCategory               string            = "category"
-	TLCategory_Entry         TelemetryCategory = "entry"
-	TLCategory_Specification TelemetryCategory = "specification"
-	TLCategory_OSLocator     TelemetryCategory = "object-store-locator"
+	TLCategory              string            = "category"
+	TlcategoryEntry         TelemetryCategory = "entry"
+	TlcategorySpecification TelemetryCategory = "specification"
+	TlcategoryOslocator     TelemetryCategory = "object-store-locator"
```

Comment:
The `c` in "category" should be capitalized for all these, e.g. `TlCategoryEntry`.